src/schema/sources.ts

```diff
@@ -1622,5 +1645,11 @@ export const resolvers: IResolvers<any, Context> = {
 
       return referralUrl;
     },
+    flags: ({ flags }: GQLSource): SourceFlagsPublic => ({
+      totalPosts: flags?.totalPosts ?? 0,
+      totalViews: flags?.totalViews ?? 0,
+      totalUpvotes: flags?.totalUpvotes ?? 0,
+      featured: isNullOrUndefined(flags?.featured) ? false : flags.featured,
```

Comment:
Also I think better to do it on graphorm level because it will be available directly in queries vs only after query resolves. Basically under `jsonType: true` with `transform`.

Reply:
https://github.com/dailydotdev/daily-api/pull/1857#discussion_r1581177349

Reply:
Yeah, that makes sense, we will have more control and avoid the null value on certain situations.